sdk/storage/perf-tests/storage-blob/package.json

```diff
@@ -8,12 +8,22 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/core-http": "^3.0.0",
-    "@azure/core-rest-pipeline": "^1.1.0",
-    "@azure/storage-blob": "^12.12.0-beta.2",
     "@azure/test-utils-perf": "^1.0.0",
-    "dotenv": "^16.0.0",
-    "uuid": "^8.3.0"
+    "dotenv": "^8.2.0",
```

Comment:
I think this is the result of running perf tests on older packages, and should be reverted.